### ISSUE TITLE
Error with Swift Package Manager (1.12.9)

### ISSUE THREAD

**Issue by knguyen2708** [rollbar/rollbar-apple#104]
If I create a new project (Xcode 12.5.1) and add Rollbar Apple as a Swift Package (currently v 1.12.9), I get the following error:

/Users/knguyen/Library/Developer/Xcode/DerivedData/Test-fabppkklxiqdukdjcjddwdpzrtxu/SourcePackages/checkouts/rollbar-apple/Rollbar/Notifier_DTOs/RollbarTrace.h:9:9: 'DataTransferObject.h' file not found

Can someone have a look?

<img width="284" alt="image" src="https://user-images.githubusercontent.com/3516271/130569608-8b0ac695-7edd-4cab-ada5-b5037167df2b.png">

**Comment by akornich** [comment]
@knguyen2708 , are you sure you reported this problem for the correct repo? I think you intended it for this one: https://github.com/rollbar/rollbar-ios
Rollbar-Apple is replacement for Rollbar-iOS SDK and starts from v2 (and it is in beta phase now - but you are more than welcome to try it). It will be moving out of bet anytime soon. Rollbar-Apple fully supports Swift Package Manager, Cocoapods, Carthage while Rollbar-iOS only supports Cocoapods and Carthage. In addition, Rollbar-Apple SDK more modular, lightweight and has more options/features than Rollbar-iOS.

Please, let me know if you need further support transitioning to Rollbar-Apple from Rollbar-iOS.

**Comment by akornich** [comment]
closing due to no response

**Comment by knguyen2708** [comment]
Hi apologies, I didn't receive notifications for this thread for some reason.

I did add this repo to Xcode, not rollbar-ios (as instructed in https://github.com/rollbar/rollbar-apple.git), but Xcode automatically resolved it to 1.12.9 -- seems it could not see the beta versions -- so I mistakenly thought SPM were supported for 1.12.9 (and there is a Package.swift file ! ). 

I have since resolved the issue by manually setting the package's version to 2.0.0 beta. 

Thanks for the suppport.
